src/website/views/custom-elements/popover.pug

```diff
@@ -0,0 +1,499 @@
+    :code(lang="html")
+      <chi-popover position="top" active arrow>
+        <div class="-px--2 -py--1 -s--3">
+          <h5>Popover!</h5>
+          <p>This is a popover content.</p>
+        </div>
+      </chi-popover>
+#a-tabs--positioning-right.a-tabs-panel
+  .example.-mb--3
+    .-p--3(style='height: 158px')
+      .-position--relative
+        chi-popover(position="right", active, arrow, prevent-auto-hide)
+          .-px--2.-py--1.-s--3
+            h5 Popover!
+            p This is a popover content.
+    :code(lang="html")
+      <chi-popover position="right" active arrow>
+        <div class="-px--2 -py--1 -s--3">
+          <h5>Popover!</h5>
+          <p>This is a popover content.</p>
+        </div>
+      </chi-popover>
+#a-tabs--positioning-bottom.a-tabs-panel
+  .example.-mb--3
+    .-p--3(style='height: 158px')
+      .-position--relative
+        chi-popover(position="bottom", active, arrow, prevent-auto-hide)
+          .-px--2.-py--1.-s--3
+            h5 Popover!
+            p This is a popover content.
+    :code(lang="html")
+      <chi-popover position="bottom" active arrow>
+        <div class="-px--2 -py--1 -s--3">
+          <h5>Popover!</h5>
+          <p>This is a popover content.</p>
+        </div>
+      </chi-popover>
+#a-tabs--positioning-left.a-tabs-panel
+  .example.-mb--3
+    .-p--3(style='height: 158px')
+      .-position--relative
+        chi-popover(position="left", active, arrow, prevent-auto-hide)
+          .-px--2.-py--1.-s--3
+            h5 Popover!
+            p This is a popover content.
+    :code(lang="html")
+      <chi-popover position="left" active arrow>
+        <div class="-px--2 -py--1 -s--3">
+          <h5>Popover!</h5>
+          <p>This is a popover content.</p>
+        </div>
+      </chi-popover>
+
+h4 Reference element
+p.-text
+  | A reference element can be setup for advanced positioning.
+  | The popover will be positioned relative to this
+  | element. Use the <code>reference</code> attribute with a
+  | CSS selector to the reference element.
+p.-text
+  span.a-badge.-primary.-mr1
+    span Tip 1
+  | This feature makes use of the Popper.js position engine.
+p.-text
+  span.a-badge.-primary.-mr1
+    span Tip 2
+  | This example uses extra JavaScript. See <a href="#show-hide-toggle">Show, hide & toggle section</a>.
+
+.example.-mb--3
+  .-p--3
+    button#example-3-button.a-btn Click me!
+    chi-popover#example-3-popover(position="top", arrow, reference="#example-3-button")
+      .-px--2.-py--1.-s--3
+        h5 Popover!
+        p This is a popover content.
+
+  :code(lang='html')
+    <button class="a-btn" id="example-3-button">Click me!</button>
+    <!-- put the following code anywere in the DOM, e.g. at the end of the DOM -->
+    <chi-popover position="top" arrow reference="example-3-button">
+      <div class="-px--2 -py--1 -s--3">
+        <h5>Popover!</h5>
+        <p>This is a popover content.</p>
+      </div>
+    </chi-popover>
+
+h4 Reference element
```

Comment:
@dani-cl-madrid this title should be positioning as it makes reference to the popover position.